.travis.yml

```diff
@@ -48,4 +48,5 @@ matrix:
     - Rscript -e 'install.packages("devtools", repos = "http://cran.rstudio.com")'
     - Rscript -e 'install.packages(c("caret","DBI","data.table","doParallel","e1071"), repos = "http://cran.rstudio.com")'
     - Rscript -e 'install.packages(c("grpreg","lme4","odbc","pROC"), repos = "http://cran.rstudio.com")'
-    - Rscript -e 'install.packages(c("R6","ranger","ROCR","RODBC","RSQLite","testthat"), repos = "http://cran.rstudio.com")'
\ No newline at end of file
+    - Rscript -e 'install.packages(c("R6","ranger","ROCR","RSQLite","testthat"), repos = "http://cran.rstudio.com")'
+    - Rscript -e 'install.packages(c("dplyr","xgboost"), repos = "http://cran.rstudio.com")'
```

Comment:
Great work! Did this dplyr addition come because of the xgboost work?

Would be amazing to not add an extra dependency (if possible)